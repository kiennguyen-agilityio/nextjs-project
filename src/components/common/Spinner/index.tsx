const Spinner = () => {
  return (
    <div
      role="status"
      className="w-4 h-4 border-4 border-t-transparent border-gray-500 rounded-full animate-spin"
    />
  );
};

export default Spinner;
