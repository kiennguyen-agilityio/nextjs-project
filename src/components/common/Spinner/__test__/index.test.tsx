import { render } from '@testing-library/react';
import Spinner from '@/components/common/Spinner';

test('renders Spinner component correctly', () => {
  const { container } = render(<Spinner />);
  expect(container).toMatchSnapshot();
});
