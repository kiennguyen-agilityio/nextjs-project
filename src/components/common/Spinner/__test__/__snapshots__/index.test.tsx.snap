// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Spinner component correctly 1`] = `
<div>
  <div
    class="w-4 h-4 border-4 border-t-transparent border-gray-500 rounded-full animate-spin"
    role="status"
  />
</div>
`;
